tests/async/test_browsertype_connect_cdp.py

```diff
@@ -30,7 +30,6 @@ async def test_connect_to_an_existing_cdp_session(
     browser_server = await browser_type.launch(
         **launch_arguments, args=[f"--remote-debugging-port={port}"]
     )
-    wait_for_port(port)
```

Comment:
We upstream wait now if the --remote-debugging-port arg is passed until the websocket server is ready.